ipamapi/contract.go

```diff
@@ -23,7 +23,7 @@ const (
 // Callback provides a Callback interface for registering an IPAM instance into LibNetwork
 type Callback interface {
 	// RegisterDriver provides a way for Remote drivers to dynamically register new NetworkType and associate with a ipam instance
-	RegisterIpamDriver(name string, driver Ipam) error
+	RegisterIpamDriver(name string, driver Ipam, capability *Capability) error
```

Comment:
This is a backward incompatible API change. I understand that this contract API will not be used directly by the real remote drivers. But still being an externally visible API, care must be taken to avoid any API breakage.